doc/source/sdk/v2/zaf_v2.md

```diff
@@ -0,0 +1,197 @@
+## Zendesk App Framework v2 Beta
+The Zendesk App Framework (ZAF) v2 introduces a new way of building Zendesk apps using iframes.
+
+The v2 Framework improves upon v1 by:
+- improving on security by taking advantage of the browsers own cross-domain security features.
+- making it easy to use whatever technologies you're familiar with.
+- providing consistent APIs across all supported Zendesk Products
+
+### How does it work?
+ZAF v2 apps include a manifest.json file and an assets folder. The manifest file includes a slightly modified locations property, which specifies URL paths for each location. These URL paths can be absolute, pointing to an external page, or relative, pointing to a html file inside the assets folder.
+
+e.g.
+```json
+"location": {
+  "zendesk": {
+    "ticket_sidebar":       "assets/iframe.html?location=ticket_sidebar",
+    "new_ticket_sidebar":   "assets/iframe.html?location=new_ticket_sidebar",
+    "user_sidebar":         "assets/user_sidebar.html",
+    "organization_sidebar": "assets/organization_sidebar.html",
+    "nav_bar":              "https://dashboard.myapp.com/nav_bar",
+    "top_bar":              "https://dashboard.myapp.com/top_bar"
+  }
+}
+```
+
+The HTML files referenced on the manifest file will need to include the ZAF SDK v2 library in a script tag. This will allow your iframe to interact with the new ZAF SDK APIs documented below.
+
+i.e.
+```html
+<script src="https://assets.zendesk.com/apps/sdk/2.0.0-beta1/zaf_sdk.js"></script>
+```
+
+### Feature parity
+While ZAF v2 aims to support all existing framework features, in some cases due to the nature of the new architecture some features may be missing or incomplete. For more information check out our feature parity status page [here]().
```

Comment:
Original:
> For more information check out our feature parity status page [here]().

Suggested edit to avoid using "here" as link text:
> For more information check out our [feature parity status page]().